### PR TITLE
Feature/alamofire5

### PR DESCRIPTION
**Pull request by carlosmouracorreia**
Possible improvement is to only have a MappableResponseSerializer (instead of 4 different classes) and have the serialize function to accept a callback parameter or have conditional logic for each of the following accepted serialisers:

- ObjectMapperSerializer
- ObjectMapperImmutableSerializer
- ObjectMapperArraySerializer
- ObjectMapperImmutableArraySerializer

### CODE REVIEW

**Comment by carlosmouracorreia** [comment]
Removed previous 4 serialisers into only 2:

ObjectMapperSerializer
ObjectMapperArraySerializer

**Comment by carlosmouracorreia** [comment]
@tristanhimmelman any possibility to check this soon? thank you!

File: AlamofireObjectMapper.podspec [hunk 1]
```diff
@@ -1,22 +1,22 @@
 Pod::Spec.new do |s|
 
   s.name = "AlamofireObjectMapper"
-  s.version = "5.2.0"
+  s.version = "5.2.1"
```
Comment (tristanhimmelman) [inline AlamofireObjectMapper.podspec]: I think should make this a major version release so we don't force users to use Alamofire 5.0.0

Comment (carlosmouracorreia) [reply AlamofireObjectMapper.podspec]: Should I change it to 5.3 or 6.0? Sorry for the delay.

Comment (tristanhimmelman) [reply AlamofireObjectMapper.podspec]: Lets go with 6.0

Comment (ios4vn) [reply AlamofireObjectMapper.podspec]: 👍

**Pull request merged**